dbt_project.yml

```diff
@@ -731,6 +731,50 @@ seeds:
             amount: double
             tx_num: long
 
+    quix:
+      +enabled: true
+      +schema: test_data
+      quix_events_seed:
+        +column_types:
+          blockchain: string
+          project: string
+          version: string
+          block_date: date
+          block_time: timestamp
+          token_id: string
+          collection: string
+          amount_usd: double
+          token_standard: string
+          trade_type: string
+          number_of_items: int
+          trade_category: string
+          evt_type: string
+          seller: string
+          buyer: string
+          amount_original: double
+          amount_raw: double
+          currency_symbol: string
+          currency_contract: string
+          nft_contract_address: string
+          project_contract_address: string
+          aggregator_name: string
+          aggregator_address: string
+          tx_hash: string
```

Comment:
Hi @chuxinh please also check if I make any mistake in column date types.

Reply:
Looks good to me, thanks!